executor/poolmgr/gpm.go

```diff
@@ -241,3 +248,59 @@ func (gpm *GenericPoolManager) IsValidPod(kubeObjects []apiv1.ObjectReference, p
 	}
 	return false
 }
+
+// idleObjectReaper reaps objects after certain idle time
+func (gpm *GenericPoolManager) idleObjectReaper(kubeClient *kubernetes.Clientset,
+	fissionClient *crd.FissionClient,
+	fsCache *fscache.FunctionServiceCache,
+	idlePodReapTime time.Duration) {
+
+	pollSleep := time.Duration(2 * time.Minute)
+	for {
+		time.Sleep(pollSleep)
+
+		envs, err := fissionClient.Environments(metav1.NamespaceAll).List(metav1.ListOptions{})
+		if err != nil {
+			log.Fatalf("Failed to get environment list: %v", err)
```

Comment:
Missing a `continue`?

Reply:
I think its intent to do so, the executor will fatal(crash) directly when it's not able to get environments objects from the API server (maybe due to the connection error, or resources not defined).